app/views/plans/_form.html.erb

```diff
@@ -30,18 +30,18 @@
         <div class="panel-body">
           <div class="form-group row">
             <div class="col-md-9">
-              <%= f.label :name, class: 'sr-only' %>
+              <%= f.label :name, class: '' %>
```

Comment:
If there's no class to be passed to the label tag, there's no need to have `class: ''`. You should remove it and do the same with other similar cases on this file.